packages/noports_core/lib/src/sshnp/impl/sshnp_unsigned_impl.dart

```diff
@@ -94,6 +95,9 @@ class SshnpUnsignedImpl extends SshnpCore with SshnpLocalSshKeyHandler {
       throw SshnpError('sshnpd did not acknowledge the request');
     }
 
+    /// Start sshrv
+    var bean = await sshrvdChannel.runSshrv(directSsh: false);
```

Comment:
uhmmm this might have been added in error, will need to check it

Reply:
Unsigned client always ran sshrv, this is the legacy v3 client. I see that it has been moved to start up after the notification for an sshd request is sent.